packages/core/package.json

```diff
@@ -86,7 +86,7 @@
     "remark-stringify": "^10.0.2",
     "unified": "^10.1.2",
     "uuid": "^8.3.2",
-    "y-prosemirror": "1.0.20",
+    "y-prosemirror": "1.2.1",
```

Comment:
I also re-generated `package-lock.json` since it seemed to be stuck on Mantine 7.4.2. This gave me an error that there was a dependency mismatch (with `@tiptap/extension-collaboration`) between 1.0.20 and 1.2.1, so this change resolved it.